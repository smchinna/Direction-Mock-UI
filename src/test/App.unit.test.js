
import React from 'react';
import enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import obj from '../all-component';

enzyme.configure({ adapter: new Adapter() });

Object.keys(obj).forEach(componentName => {
  const Component = obj[componentName];
  describe(`Component: ${componentName}`, () => {
    test(`${componentName} renders with default props`, () => {
      const wrapper = shallow(<Component />);
      expect(wrapper).toMatchSnapshot();
    });
  });
}); 
