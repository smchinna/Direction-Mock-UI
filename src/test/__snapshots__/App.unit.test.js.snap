// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: App App renders with default props 1`] = `ShallowWrapper {}`;

exports[`Component: Button Button renders with default props 1`] = `ShallowWrapper {}`;

exports[`Component: CheckBox CheckBox renders with default props 1`] = `ShallowWrapper {}`;

exports[`Component: InputCom InputCom renders with default props 1`] = `ShallowWrapper {}`;

exports[`Component: ModalPopup ModalPopup renders with default props 1`] = `ShallowWrapper {}`;

exports[`Component: Panel Panel renders with default props 1`] = `ShallowWrapper {}`;

exports[`Component: PanelBody PanelBody renders with default props 1`] = `ShallowWrapper {}`;

exports[`Component: PanelHeader PanelHeader renders with default props 1`] = `ShallowWrapper {}`;

exports[`Main App Render without Cash 1`] = `ShallowWrapper {}`;
